docs/build_from_source.md

```diff
@@ -0,0 +1,98 @@
+# Building from source
+
+This document gives information how to build docker images from source with different variants.
+
+## Prerequisites
+
+1. [Docker Engine](https://docs.docker.com/engine/)
+1. Ubuntu 20.04 or RedHat 8.7 host
+1. make
+1. bash
+
```

Comment:
1., 1., 1., 1.?

Reply:
This is intended, md format is able to convert it into 1,2,3,4.

Reply:
View generated md file

Reply:
https://github.com/openvinotoolkit/model_server/blob/build-from-src/docs/build_from_source.md